README.md

```diff
@@ -0,0 +1,84 @@
+# Clearable Token
+
+[![Build Status](https://travis-ci.org/IoBuilders/clearable-token.svg?branch=master)](https://travis-ci.org/IoBuilders/clearable-token)
+[![Coverage Status](https://coveralls.io/repos/github/IoBuilders/clearable-token/badge.svg?branch=master)](https://coveralls.io/github/IoBuilders/clearable-token?branch=master)
+[![npm](https://img.shields.io/npm/v/eip1996.svg)](https://www.npmjs.com/package/eip1996)
+
+This is the reference implementation of [EIP Clearable Token](https://github.com/IoBuilders/EIPs/blob/eip-clearable-token/EIPS/eip-clearable-token.md). This implementation will change over time with the token standard and is not stable at the moment.
```

Comment:
The link should point to the standard on the Ehtereum EIP repository: https://github.com/ethereum/EIPs/pull/2018/files